include/Core/UI/Setting/EmailLogSetting.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace EmailLog\Core\UI\Setting;
+use EmailLog\Core\UI\Setting\SettingField;
+
+class EmailLogSetting extends Setting {
+
+	/**
```

Comment:
The phpdoc block for this method and the other ones in this class should be changed.

Reply:
Do you mean to add the Doc block only to the Class here?

Reply:
Yeah. If the methods in the parent class already have a phpdoc block and in the child class we are just inheriting the class without any additional functionality (especially in the case of abstract methods) then we can ignore the phpdoc block for the inherited methods.

phpdoc will automatically inherit documentation.

We need to add phpdoc block in inherited methods only if we are going to have a different phpdoc block or if we have to add any child class specific information.

Reply:
> We need to add phpdoc block in inherited methods only if we are going to have a different phpdoc block or if we have to add any child class specific information.

Understood!